### PR TITLE
fix: urbanists and laborists have opinions on economic system if decentralized

### PR DESCRIPTION
**Pull request by Mercur-3**
nations that start with "decentralized" economic system now have 'social" urbanists and laborists

### CODE REVIEW

File: common/on_actions/00_code_on_actions.txt [hunk 1]
```diff
@@ -1272,12 +1272,21 @@ on_law_enactment_pass = {
...
 							not = {
 								has_law = law_type:law_interventionism
+								has_law = law_type:law_decentralization
 							}
 						}
 						and = {
 							currently_enacting_law.type = law_type:law_interventionism
 							not = {
 								has_law = law_type:law_free_market
+								has_law = law_type:law_dec
```
Comment (JackEPepper) [inline common/on_actions/00_code_on_actions.txt:1282]:
```suggestion
								has_law = law_type:law_decentralization
```

File: common/on_actions/00_code_on_actions.txt [hunk 2]
```diff
@@ -1345,12 +1354,21 @@ on_law_enactment_pass = {
...
```
Comment (JackEPepper) [inline common/on_actions/00_code_on_actions.txt:1364]:
```suggestion
								has_law = law_type:law_decentralization
```

**Review by JackEPepper — CHANGES_REQUESTED** [review]

**Comment by JackEPepper** [comment]
Need to change this branch to dev

**Comment by DZappala** [comment]
> Need to change this branch to dev

done

**Review by JackEPepper — DISMISSED** [review]

File: common/on_actions/00_code_on_actions.txt [hunk 3]
```diff
@@ -1272,12 +1272,21 @@ on_law_enactment_pass = {
 							currently_enacting_law.type = law_type:law_free_market
 							not = {
```
Comment (DZappala) [inline common/on_actions/00_code_on_actions.txt:1273]:
```suggestion
							nor = {
```

File: common/on_actions/00_code_on_actions.txt [hunk 4]
```diff
@@ -1272,12 +1272,21 @@ on_law_enactment_pass = {
...
 								has_law = law_type:law_interventionism
+								has_law = law_type:law_decentralization
 							}
 						}
 						and = {
 							currently_enacting_law.type = law_type:law_interventionism
 							not = {
```
Comment (DZappala) [inline common/on_actions/00_code_on_actions.txt:1280]:
```suggestion
							nor= {
```

File: common/on_actions/00_code_on_actions.txt [hunk 5]
```diff
@@ -1272,12 +1272,21 @@ on_law_enactment_pass = {
...
 								has_law = law_type:law_free_market
+								has_law = law_type:law_decentralization
+							}
+						}
+						and = {
+							currently_enacting_law.type = law_type:law_decentralization
+							not = {
```
Comment (DZappala) [inline common/on_actions/00_code_on_actions.txt:1287]:
```suggestion
							nor = {
```

File: common/on_actions/00_code_on_actions.txt [hunk 6]
```diff
@@ -1345,12 +1354,21 @@ on_law_enactment_pass = {
 							currently_enacting_law.type = law_type:law_free_market
 							not = {
```
Comment (DZappala) [inline common/on_actions/00_code_on_actions.txt:1355]:
```suggestion
							nor = {
```

File: common/on_actions/00_code_on_actions.txt [hunk 7]
```diff
@@ -1345,12 +1354,21 @@ on_law_enactment_pass = {
...
 								has_law = law_type:law_interventionism
+								has_law = law_type:law_decentralization
 							}
 						}
 						and = {
 							currently_enacting_law.type = law_type:law_interventionism
 							not = {
```
Comment (DZappala) [inline common/on_actions/00_code_on_actions.txt:1362]:
```suggestion
							nor = {
```

File: common/on_actions/00_code_on_actions.txt [hunk 8]
```diff
@@ -1345,12 +1354,21 @@ on_law_enactment_pass = {
...
 								has_law = law_type:law_free_market
+								has_law = law_type:law_decentralization
+							}
+						}
+						and = {
+							currently_enacting_law.type = law_type:law_decentralization
+							not = {
```
Comment (DZappala) [inline common/on_actions/00_code_on_actions.txt:1369]:
```suggestion
							nor = {
```

**Review by DZappala — CHANGES_REQUESTED** [review]

**Review by DZappala — APPROVED** [review]

**Pull request merged**